metadata.rb

```diff
@@ -8,7 +8,11 @@
 
 supports         'centos', '~> 7.0'
 
+depends          'build-essential'
 depends          'firewall'
+depends          'git'
+depends          'poise-python'
+depends          'nodejs'
```

Comment:
nodejs is required for rails and we might as well install python for future python apps.